mm2src/coins/lightning_persister/src/lib.rs

```diff
@@ -74,10 +89,521 @@ where
+            last_updated INTEGER NOT NULL
+        );",
+        table_name
+    );
+
+    Ok(sql)
+}
+
+fn create_payments_history_table_sql(for_coin: &str) -> Result<String, SqlError> {
+    let table_name = payments_history_table(for_coin);
+    validate_table_name(&table_name)?;
+
+    let sql = format!(
+        "CREATE TABLE IF NOT EXISTS {} (
+            id INTEGER NOT NULL PRIMARY KEY,
+            payment_hash VARCHAR(255) NOT NULL UNIQUE,
+            destination VARCHAR(255),
+            description VARCHAR(641) NOT NULL,
+            preimage VARCHAR(255),
+            secret VARCHAR(255),
+            amount_msat INTEGER,
+            fee_paid_msat INTEGER,
+            is_outbound INTEGER NOT NULL,
+            status VARCHAR(255) NOT NULL,
+            created_at INTEGER NOT NULL,
+            last_updated INTEGER NOT NULL
+        );",
+        table_name
+    );
+
+    Ok(sql)
+}
+
+fn insert_channel_sql(for_coin: &str) -> Result<String, SqlError> {
+    let table_name = channels_history_table(for_coin);
+    validate_table_name(&table_name)?;
+
+    let sql = format!(
+        "INSERT INTO {} (
+            rpc_id,
+            channel_id,
+            counterparty_node_id,
+            is_outbound,
+            is_public,
+            is_closed,
+            created_at,
+            last_updated
+        ) VALUES (
+            ?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8
+        );",
+        table_name
+    );
+
+    Ok(sql)
+}
+
+fn insert_or_update_payment_sql(for_coin: &str) -> Result<String, SqlError> {
+    let table_name = payments_history_table(for_coin);
+    validate_table_name(&table_name)?;
+
+    let sql = format!(
+        "INSERT OR REPLACE INTO {} (
+            payment_hash,
+            destination,
+            description,
+            preimage,
+            secret,
+            amount_msat,
+            fee_paid_msat,
+            is_outbound,
+            status,
+            created_at,
+            last_updated
+        ) VALUES (
+            ?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11
+        );",
+        table_name
+    );
+
+    Ok(sql)
+}
+
+fn select_channel_from_table_by_rpc_id_sql(for_coin: &str) -> Result<String, SqlError> {
+    let table_name = channels_history_table(for_coin);
+    validate_table_name(&table_name)?;
+
+    let sql = format!(
+        "SELECT 
+            rpc_id,
+            channel_id,
+            counterparty_node_id,
+            funding_tx,
+            funding_value,
+            funding_generated_in_block,
+            closing_tx,
+            closure_reason,
+            claiming_tx,
+            claimed_balance,
+            is_outbound,
+            is_public,
+            is_closed,
+            created_at,
+            last_updated
+        FROM
+            {}
+        WHERE
+            rpc_id=?1",
+        table_name
+    );
+
+    Ok(sql)
+}
+
+fn select_payment_from_table_by_hash_sql(for_coin: &str) -> Result<String, SqlError> {
```

Comment:
Do we need to include `from_table` in function names, since it clearly has `sql` end of it. Excluding `from_table` from the sql functions can help us to decrease function names. They look quite long :slightly_smiling_face: